package.json

```diff
@@ -1,8 +1,40 @@
 {
   "name": "bats",
-  "version": "v1.0.1",
+  "version": "0.4.0",
   "description": "Bash Automated Testing System",
   "homepage": "https://github.com/bats-core/bats-core#readme",
-  "repository": "github:bats-core/bats-core",
-  "bin": "./bin/bats"
+  "license": "MIT",
+  "author": "Sam Stephenson <sstephenson@gmail.com> (http://sstephenson.us/)",
+  "contributors": [
```

Comment:
Would an AUTHORS file be more desirable? Just got the idea from https://docs.npmjs.com/files/package.json#default-values.

Reply:
Yeah, I'm cool with that. Generally I try to minimize files in repo root. But since AUTHORS would (presumably) support non-node/npm tools, I'm 👍